code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -200,10 +207,23 @@
 	alert_on_death = TRUE
 	melee_block_chance = 60
 
+/mob/living/simple_animal/hostile/syndicate/melee/autogib/depot/officer/Initialize()
+	..()

```

Comment:
need to do a non-explicit return here that still returns parent value.

Reply:
Done